### ISSUE TITLE
[BUG] New comments not showing up in the comment feed. Only after refreshing the browser

### ISSUE THREAD

**Issue by JulieMass** [dev-launchers/dev-launchers-platform#835]
### Steps to replicate are

1. Go to IdeaSpace > Browse Ideas https://devlaunchers.org/ideas/browse 
2. Click “See more” to access an idea’s workshopping page 
3. Log in to create a new comment
4. Create a new comment

### Device and OS
Desktop & mobile, Browsers: Chrome & Microsoft Edge
### Expected behavior
New comments should show up immediately after the user submits it without having to refresh the browser.
### Actual behavior
While submitting a new comment it doesn’t immediately shows up in the comment feed which can create confusion for people not knowing why their comments aren’t showing up.

![image.png](https://images.zenhubusercontent.com/63400f3d1e79614e45d9ebd9/9887e926-b0f0-4779-8d15-0a7574eefd88)

**Comment by JulieMass** [comment]
### Steps to replicate are
Go to IdeaSpace > Browse Ideas https://devlaunchers.org/ideas/browse 
Click “See more” to access an idea’s workshopping page 
Log in to create a new comment
Create a new comment
### Device and OS
Desktop & mobile, Browsers: Chrome & Microsoft Edge
### Expected behavior
New comments should show up immediately after the user submits it without having to refresh the browser.
### Actual behavior
While submitting a new comment it doesn’t immediately shows up in the comment feed which can create confusion for people not knowing why their comments aren’t showing up. 

![image.png](https://images.zenhubusercontent.com/63400f3d1e79614e45d9ebd9/9939d24e-367f-48c2-a028-2775524f9eda

**Comment by JulieMass** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/IdeaSpace-Team-6350185d9843157427cc900b/issues/dev-launchers/dev-launchers-platform/835?planning-poker) with Zenhub @alicefate @dbradham @jordanmcousins @JulieMass @Nouri-Anouar @RETprojects

**Comment by JulieMass** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/IdeaSpace-Team-6350185d9843157427cc900b/issues/dev-launchers/dev-launchers-platform/835?planning-poker) with Zenhub @rashikashaw

**Comment by JulieMass** [comment]
Question for @RETprojects, @jordanmcousins and @dbradham 

During our weekly meeting yesterday, Remi mentioned that he had fixed the issue of posts appearing immediately without the need for the page to be refreshed (which is a GREAT news by the way! YAY! Thanks Remi 👏👏!). I recalled this bug that had been on hold since April, which Jordan was working on, and I was thinking: **should we close this bug ticket and consider it resolved since Remi has addressed it with his ticket #748 for the comment feed?** What do you all think about this?

**Comment by RETprojects** [comment]
Hi @JulieMass! I think we can resolve this bug ticket now that the new comments appear immediately.

**Comment by JulieMass** [comment]
> Hi @JulieMass! I think we can resolve this bug ticket now that the new comments appear immediately.

Awesoooomeeeee! Thanks Remi!

![giphy](https://github.com/dev-launchers/dev-launchers-platform/assets/104226366/17fe2720-0cca-4bd6-89d1-971b1949ac24)
